### DISCUSSION TITLE
Chia plot not going past %20

### DISCUSSION THREAD

**Discussion by FrozenOwl1** [Chia-Network/chia-blockchain#8446] [General]
I started mining chia and my plot won't go past %20 and on computation table 4. my final destination drive has been stuck at 100% for the last day and a half.

**Comment by loppefaaret** [comment]
if you open up the log, what does the last few lines show ?

**Comment by BrandtH22** [comment]
@FrozenOwl1 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
